ccd-definition/CaseEvent/Gatekeeping.json

```diff
@@ -118,13 +118,29 @@
     "ShowEventNotes": "Y",
     "EndButtonLabel": "Save and continue"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "createNoticeOfProceedingsGatekeeping",
+    "Name": "Create notice of proceedings",
+    "Description": "Create notice of proceedings",
+    "DisplayOrder": 9,
```

Comment:
is the display order correct?

Reply:
To the best of my knowledge yes -  I don't think the display order for events has been properly revised. Not too important imo, probably needs a discussion later once we have all events in system? easy to swap display orders

Reply:
<img width="252" alt="Screenshot 2019-11-25 at 12 49 33" src="https://user-images.githubusercontent.com/15530123/69542119-a795d100-0f82-11ea-98fd-5a50296f2be1.png">
<img width="254" alt="Screenshot 2019-11-25 at 12 49 10" src="https://user-images.githubusercontent.com/15530123/69542125-acf31b80-0f82-11ea-8431-9b2a926eba7d.png">
You can see here submitted vs gatekeeping event list. Fairly inconsistent! We should put it in a separate story later to make these consistent, not scope of this story to other fix display orders